tox.ini

```diff
@@ -81,6 +81,8 @@ envlist =
 
     {py3.6,py3.7,py3.8}-chalice-{1.16,1.17,1.18,1.19,1.20}
 
+    {py2.7,py3.7,py3.8}-boto3-{1.16}
+
 [testenv]
 deps =
```

Comment:
I think you will have to add `boto3: boto3` (with a version constraint pls) here to make CI work